### PR TITLE
feature: Add field(type) for project

### PR DESCRIPTION
**Pull request by innerpeacez**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @liuhaoyang @recallsong 

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.2` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/core-services/model/project.go [hunk 1]
```diff
@@ -20,21 +20,22 @@ import (
...
+	RollbackConfig string // Rollback configuration: {"DEV": 1,"TEST": 2,"STAGING": 3,"PROD": 4}
 	CpuQuota       float64
 	MemQuota       float64
 	Functions      string    `gorm:"column:functions"`
 	ActiveTime     time.Time `gorm:"column:active_time"`
-	EnableNS       bool      `gorm:"column:enable_ns"` // 是否打开项目级命名空间
-	IsPublic       bool      `gorm:"column:is_public"` // 是否是公开项目
+	EnableNS       bool      `gorm:"column:enable_ns"` // Whether to open the project-level namespace
+	IsPublic       bool      `gorm:"column:is_public"` // Is it a public project
+	TYPE           string    `gorm:"column:type"`      // project type
```
Comment (recallsong) [inline modules/core-services/model/project.go:38]: Type

Comment (liuhaoyang) [inline modules/core-services/model/project.go:38]: Use camel case for naming, with initial capital letters, like `Type`

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**